src/elements/rich-link/RichlinkForm.tsx

```diff
@@ -30,5 +58,14 @@ export const RichlinkElementForm: React.FunctionComponent<Props> = ({
       errors={errors.weighting}
       display="inline"
     />
+    <div css={warningContainer}>
```

Comment:
Could we put the conditional one level up, so there's no warning markup at all unless we have an alert?

Reply:
> put the conditional one level up, so there's no warning markup

Really good spot @jonathonherbert the mark up was definitely there even when no alert needed. I have moved the conditional to contain the parent <div>